RHIDP-5422: TechDocs add-ons doc (#975)
redhat-developer/red-hat-developers-documentation-rhdh: modules/techdocs/proc-techdocs-addon-remove-operator.adoc

. From the *ConfigMaps* page, click the ConfigMap that contains the TechDocs add-on that you want to remove.
. Select the *YAML view* option in the *Configure via* field.
. In the `plugins` section of the ConfigMap, do one of the following actions based on whether you want to disable or remove a TechDocs add-on:
** To temporarily disable all of the TechDocs add-ons in a particular plugin package, change the value of the `disabled` field to `true`. For example:
+
[source,yaml,subs="+quotes,+attributes"]
----
kind: ConfigMap
apiVersion: v1
metadata:
  name: dynamic-plugins-rhdh
data:
  dynamic-plugins.yaml: |
    includes:
      - dynamic-plugins.default.yaml
    plugins:
      - package: ./dynamic-plugins/dist/backstage-plugin-techdocs-module-addons-contrib
        disabled: true
        pluginConfig:
          dynamicPlugins:
            frontend:
              backstage.plugin-techdocs-module-addons-contrib:
                techdocsAddons:
                  - importName: ReportIssue
                  - importName: _<external_techdocs_add-on>_
----
+
where:

_<external_techdocs_add-on>_:: Specifies the external TechDocs add-on that you want to remove, for example, `TextSize`.

> ** To permanently remove one or more TechDocs add-ons from a plugin package, delete `importName: _<external_techdocs_add-on>` for each external TechDocs add-on that you want to remove. For example:

Reviewer: ```suggestion
** To remove one or more TechDocs add-ons from use in {product}, delete `importName: _<external_techdocs_add-on>_` for each external TechDocs add-on that you want to remove. For example:
```
